mitodl/open-discussions
# Track user activity

#### What are the relevant tickets?
Fixes #473 

Adds user tracking middleware and refactors and upgrades existing middleware to use `MIDDLEWARES` style

#### How should this be manually tested?
Navigate around open discussions. You should see the JWT cookie updated on the first page/API request made to the server (subject to thundering herd, but shouldn't cause errors). You set see no further attempts to `Set-Cookie` unless you get a new JWT token.
